api/controllers/subscriber.js

```diff
@@ -380,10 +380,11 @@ async function createTransaction(req, res) {
         });
       }
       const transaction = subscriber.transaction;
+      const id = (transaction.platform === 'android-playstore' && transaction.nativePurchase.productId) || (transaction.platform === 'paypal' && transaction.products[0].id);
```

Comment:
this is a logical operation, it will always return `false` or `true`, but  never the id